src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    flex-direction: row;
+    display: flex;
+    width: 16px;
+    height: 16px;
+    flex-shrink: 0;
+    align-items: center;
+    margin-right: 4px;
+    background: url("/images/star.svg") no-repeat  center center;
+    z-index: 1;
```

Comment:
why do you need z-index?

Reply:
So that when i change modifier, active-stars cover regular ones